tests/testsuite/dotnet.rs

```diff
@@ -12,12 +12,39 @@ fn shows_nothing_in_directory_with_zero_relevant_files() -> io::Result<()> {
     workspace.close()
 }
 
+#[test]
+#[ignore]
+fn shows_latest_in_directory_with_directory_build_props_file() -> io::Result<()> {
+    let workspace = create_workspace(false)?;
+    touch_path(&workspace, "Directory.Build.props", None)?;
+    expect_output(&workspace, ".", Some("•NET v"))?;
+    workspace.close()
+}
+
+#[test]
+#[ignore]
+fn shows_latest_in_directory_with_directory_build_targets_file() -> io::Result<()> {
+    let workspace = create_workspace(false)?;
+    touch_path(&workspace, "Directory.Build.targets", None)?;
+    expect_output(&workspace, ".", Some("•NET v"))?;
+    workspace.close()
+}
+
+#[test]
+#[ignore]
+fn shows_latest_in_directory_with_packages_props_file() -> io::Result<()> {
+    let workspace = create_workspace(false)?;
+    touch_path(&workspace, "Packages.props", None)?;
+    expect_output(&workspace, ".", Some("•NET v"))?;
+    workspace.close()
+}
+
 #[test]
 #[ignore]
 fn shows_latest_in_directory_with_solution() -> io::Result<()> {
     let workspace = create_workspace(false)?;
     touch_path(&workspace, "solution.sln", None)?;
-    expect_output(&workspace, ".", Some("•NET v2.2.402"))?;
+    expect_output(&workspace, ".", Some("•NET v"))?;
```

Comment:
While I now understand why these changes were necessary/correct, I spent about 30 minutes being very confused over what was going on here.

Could we leave a comment (either at the top of the dotnet test module or at the first time it's used) explaining why this form is the correct output to expect, and isn't just us breaking the tests so that the version doesn't need to display?

Alternatively, is there a way we could make this match better, e.g. by employing a regex in testing so that we at least know we're getting valid version numbers on the end and not random strings?

Reply:
i could add a regex to this to make it more robust. i'm dabbling for the first time with rust for this so it'll take me a bit.

Reply:
@chipbuster great thanks! i just refactored to account for a pattern string to being passed to the `expect_output` function. ill get the formatting fixed in a jiffy